app/platform/platform.c

```diff
@@ -859,3 +859,8 @@ uint32_t platform_flash_mapped2phys (uint32_t mapped_addr)
   uint32_t meg = (b1 << 1) | b0;
   return mapped_addr - INTERNAL_FLASH_MAPPED_ADDRESS + meg * 0x100000;
 }
+
+void* platform_print_deprecation_note( const char *msg, const char *time_frame)
```

Comment:
I wouldn't want to keep track of whether a particular `msg` has already been printed or not. Besides, printing once only isn't enough IMO.